tests/rc_string_ut/rc_string_mocked.c

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+
+#define vsnprintf mocked_vsnprintf
+
+extern int mocked_vsnprintf(char* s, size_t n, const char* format, ...);
```

Comment:
> ...); [](http://example.com/codeflow?start=66&length=6)

The signature of vsnprintf per C99 is:
int vsnprintf(char * restrict s, size_t n,
const char * restrict format,
va_list arg);